bedrock/firefox/templates/firefox/whatsnew/exp/whatsnew-fx70-5.html

```diff
@@ -0,0 +1,99 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import fxa_cta_link with context %}
+
+{% add_lang_files "firefox/whatsnew_70" %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ _('What’s new with Firefox - More privacy, more protections.') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_70_experiment') }}
+{% endblock %}
+
+{% block body_class %}state-fxa-default{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew70' %}
+{% set _utm_campaign = 'whatsnew70' %}
+{% set _entrypoint_variation = '5' %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox mzp-t-dark">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success show-firefox-desktop-70">
+        <p>{{ _('Congrats! You’re using the latest version of Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+      <div class="mzp-c-emphasis-box">
+        <img class="c-emphasis-box-logo" src="{{ static('protocol/img/logos/firefox/logo-md.png') }}" alt="">
+        <h2 class="c-emphasis-box-title">
+          {{ _('10 billion trackers blocked is 10 billion reasons to join Firefox') }}
+        </h2>
+        <p>{{ _('The Firefox browser keeps your internetting safe from invasive companies and advertisers. Now get the same respect for your privacy in the rest of your online life.') }}</p>
+
+        <p class="cta">{{ fxa_cta_link(entrypoint=_entrypoint, entrypoint_experiment=_utm_campaign, entrypoint_variation=_entrypoint_variation, action='signup', button_text='Sign In', button_class='mzp-c-button mzp-t-product mzp-t-small', utm_campaign=_utm_campaign, utm_content='top-cta') }}</p>
```

Comment:
Yeah, we discussed this and wanted to avoid complicating this test with state-based conditional content (even though it's not _that_ hard, it was just an easy corner to cut). If you click "Sign In" without an account you'll be prompted to make one, and the end of that still results in signing in, so we can accept that for purposes of this test. If one of these variants becomes a winner for the next WNP I think we'll want to change the button based on user state.